### PR TITLE
Add security configs

### LINKED ISSUE #323: Spring security configs
Hello!
Please copy the spring security configurations from account api service to this project for decrypt information about a user from access token.

That's all.
Thank you.

### PR DESCRIPTION
**Pull request by camelya58**
Closes #323

### CODE REVIEW

**Review by daria80 — APPROVED** [review]

**Review by egnaf — APPROVED** [review]
ok

**Review by Marina2302 — APPROVED** [review]

File: config/checkstyle/checkstyle.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <module name="FileLength"><property name="max" value="1340"/></module>
 
     <module name="TreeWalker">
-        <module name="MethodLength"><property name="max" value="60"/></module>
+        <module name="MethodLength"><property name="max" value="62"/></module>
```
Comment (Hardelele) [inline config/checkstyle/checkstyle.xml:9]: What a heck) fraud!

**Review by Hardelele — APPROVED** [review]
In the future DefaultTokenServices could be necessary to replace. In some places it is hard-coded. And I would pay attention to TokenStore. Potentially problematic.

**Comment by camelya58** [comment]
> In the future DefaultTokenServices could be necessary to replace. In some places it is hard-coded. And I would pay attention to TokenStore. Potentially problematic.

I agree. I've just wanted to check the work of token services. But we could delete all info connecting with DefaultTokenServices and leave only Token Store methods in controller.
What do you think?

**Pull request merged**